Gemfile

```diff
@@ -20,6 +20,8 @@ gem 'haml-rails'
 gem 'react-rails'
 gem 'redcarpet'
 
+gem "rails-settings-cached", "~> 2.0"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5443028)